kasa/tests/test_bulb.py

```diff
@@ -134,39 +135,39 @@ async def test_variable_temp_state_information(dev: SmartBulb):
 
 @variable_temp
 @turn_on
-async def test_try_set_colortemp(dev: SmartBulb, turn_on):
+async def test_try_set_colortemp(dev: Bulb, turn_on):
     await handle_turn_on(dev, turn_on)
     await dev.set_color_temp(2700)
     await dev.update()
     assert dev.color_temp == 2700
 
 
 @variable_temp_iot
```

Comment:
I think we will want to move any protocol specific tests inside their own test modules (if not even creating a package per family?), but that can wait for another cleanup PR.